resources/fetchers/file_system_fetcher.go

```diff
@@ -40,18 +41,23 @@ const (
 	GroupFile      = "/hostfs/etc/group"
 )
 
-type FileSystemResource struct {
+type EvalFSResource struct {
 	Name    string `json:"name"`
 	Mode    string `json:"mode"`
-	Gid     uint32 `json:"gid"`
-	Uid     uint32 `json:"uid"`
+	Gid     string `json:"gid"`
```

Comment:
I see here schema change which raises some concerns for me that I wanna make sure we've covered.

1. We should check if this breaks any existing elasticsearch mapping
2. In case this is the format of the input for the OPA rules, we need to check it is being evaluated correctly.

Reply:
@kfirpeled 
1. The raw resource isn't being mapped [(index_mapping)](https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/data_stream/findings/fields/findings.yml#L48-L50)
2. Currently, we don't use `Uid` & `Gid` in our OPA code.